api/app/db/models/thessian_polygon_areas.py

```diff
@@ -0,0 +1,19 @@
+""" Class models that reflect resources and map to database tables for HFI Calculator.
+"""
+from geoalchemy2.types import Geometry
+from sqlalchemy import (Column, Integer,
+                        Sequence)
+from app.db.database import Base
+
+
+class ThessianPolygonArea(Base):
```

Comment:
Pedantic, but classname differs from tablename.

Reply:
The source feature has some properties we should probably bring in. Most pertinent/only one I think we need is the station code.